bode/bode/models/task_relations_actions.py

```diff
@@ -0,0 +1,57 @@
+from bode.app import db
```

Comment:
Nice idea to refactor this. I'm not sure if it should be in the `models` directory (because it doesn't actually contain any models) but I think we can leave it as it is for now